pkg/engine/deployment.go

```diff
@@ -242,7 +271,8 @@ func newDeployment(ctx *Context, info *deploymentContext, opts *deploymentOption
 			}
 		}
 
-		depl, err = deploy.NewImportDeployment(plugctx, target, proj.Name, opts.imports, dryRun)
+		depl, err = deploy.NewImportDeployment(
+			plugctx, deplOpts, actions, target, proj.Name, opts.imports)
```

Comment:
I assume it's correct to pass `actions` to a `NewImportDeployment` too? Or would `nil`/something with just event handlers be a better choice?